Content.Server/_CD/Admin/Commands/EventPreferencesCommand.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System.Linq;
+using Content.Server.Administration;
+using Content.Server.EUI;
+using Content.Server.Preferences.Managers;
+using Content.Shared.Administration;
+using Content.Shared.Preferences;
+using Robust.Server.Player;
+using Robust.Shared.Console;
+
+namespace Content.Server._CD.Admin.Commands;
+
+[AdminCommand(AdminFlags.Admin)]
+public sealed class EventPreferencesCommand : LocalizedCommands
```

Comment:
Toolshed commands are preferred to legacy commands because legacy commands are deprecated upstream, however this is not a blocker on this PR as I don't expect them to be removed any time in the foreseeable future.

Reply:
there's also the fact that a lot of admins do not know how to work toolshed from when i last remember, so very much won't be getting removed soon, will look into making it a toolshed command in a future pr however